src/main/java/org/opensearch/security/privileges/UserAttributes.java

```diff
@@ -0,0 +1,69 @@
+/*
```

Comment:
minor comment: As a TODO, we can now add specific UTs for this class.

Reply:
Lets not block merging for these follow ups